### DISCUSSION TITLE
Can NetAlertX monitor external network?

### DISCUSSION THREAD

**Discussion by Truepuppyicecurrant** [netalertx/NetAlertX#915] [General]
I have a VPN setup between my home and my office. I was wondering if I am able to setup NetAlertX on my home network and for it to monitor both networks (home and office). The VPN is setup on the two routers.

**Comment by jokob-sk** [comment]
Hey @Truepuppyicecurrant ,

Arpscan can't traverse VPNs (they work on different network layers). However you can use other plugins if you can get them working of if your routers support them (SNMP, DHCPLSS...):

https://github.com/jokob-sk/NetAlertX/blob/main/front/plugins/README.md

An alternative is to set up 2 NAX instances and sync them with the SyncHub plugin. 

More info on how to scan different subnets check this:

https://github.com/jokob-sk/NetAlertX/blob/main/docs/SUBNETS.md

**Reply by jokob-sk** [reply]
Oh - in the next version I'll be adding an ICMP plugin and the ability to add devices manually, so you could scan the devices that way.

**Reply by jokob-sk** [reply]
I also wrote up this guide: https://github.com/jokob-sk/NetAlertX/blob/main/docs/REMOTE_NETWORKS.md the FAKE_MAC setting will be available in the next release or you can test it in the `netalertx-dev` container
